students/Kenneth_Murray/lesson3/mailbox1/mailroom.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+import datetime
+
+"""Contribution Mail Room"""
+"""Data structure that holds a list of your donors and a history of the amounts they have donated"""
+
+contributor_list = [('Ken','Murray','10.00', '4.00'),('Tew','Tangsuk','2.68','78268','265.68'),('Joe','joe','5.50','57.89'),('Tina','Tangsuk','29.02'),('Nathan','Merrill','20.00')]
+
+
+def donor_list():
+    """Returns a list of donors sorted by first name"""
+    names_of_donors = []
+    for name in contributor_list:
+        names_of_donors.append(name[0]+" "+name[1])
+    return names_of_donors
+
+def add_donor(name):
+    """Adds a first and last name to the list"""
+    contributor_list.append(tuple(name.split(" ")))
+    return contributor_list
+
+def is_donor(check_name):
+    """Return true or false"""
+    names_of_donors = []
+    for name in contributor_list:
+        names_of_donors.append(name[0]+" "+name[1])
+    for fullname in names_of_donors:
+        if fullname.casefold() == check_name.casefold():
+            return "true"
+
+
+def new_donation(donation,donor_name):
+    """Adds a new donation to the contribution list"""
+    fullname = tuple(donor_name.split(" "))
+    firstname = fullname[0]
+    lastname = fullname[1]
+    counter = 0
+    while counter <= len(contributor_list) - 1:
+        if (contributor_list[counter])[0] == firstname and (contributor_list[counter])[1] == lastname:
+            index = counter
+            contributor_list[index] = contributor_list[index] + (str(donation),)
+        counter = counter + 1
+
+
+def thankyou_email(donation,donor_name):
+    print(f"Dear {donor_name}:")
+    print()
+    print()
```

Comment:
instead of all these `prints` use `\n`
```
print(f"Dear {donor_name}:\n\nThank you for your donation of ${donation}. Your gift is greatly appreciated.\nYour generosity and thoughtfulness means so much to all of us\n\nSincerely,\nKenneth Murray
```")